.github/workflows/unit-tests.yml

```diff
@@ -64,7 +64,7 @@ jobs:
 
           echo COLUMNS=120 >> $GITHUB_ENV
           echo PHPUNIT="$(pwd)/phpunit --exclude-group tty,benchmark,intl-data,integration" >> $GITHUB_ENV
-          echo COMPOSER_UP='composer update --no-progress --ansi'$([[ "${{ matrix.php }}" = "8.2" ]] && echo ' --ignore-platform-req=php+') >> $GITHUB_ENV
+          echo COMPOSER_UP='composer update --no-progress --ansi'$([[ "${{ matrix.php }}" = "8.3" ]] && echo ' --ignore-platform-req=php+') >> $GITHUB_ENV
```

Comment:
shouldn't this change about using `--ignore-platform-req=php+` be done in older branches as well now that PHP 8.2 is stable ?
And maybe we should base it on `matrix.mode = "experimental"` instead, to avoid having to update it regularly.